.env

```diff
@@ -1,5 +1,7 @@
-ISSUER_SEED=SDBNBVVCS6OS4VB7Z7DUETAZJUCNRYQXZQNIBZOTLE3ZX44PUUPTU2L7
-ISSUER_ADDRESS=GCHL4DBMIBZE44WLDR6LZCPWOX72TVKXSYZ65MLOFB24FUZGQCXOXMOT
-DISTRIBUTOR_SEED=SDBNBVVCS6OS4VB7Z7DUETAZJUCNRYQXZQNIBZOTLE3ZX44PUUPTU2L7
-DISTRIBUTOR_ADDRESS=GAQHJNQOW2TC2QMRWH2UEZ2QIB2BDMNPOMOQPUFQ7E5CT4RYDZYRG235
-RECIPIENT_ADDRESS=GCHL4DBMIBZE44WLDR6LZCPWOX72TVKXSYZ65MLOFB24FUZGQCXOXMOT
+# BTCT issuer for testnet.
```

Comment:
This is a public gem -- while these are just test accounts someone could screw up our testing if they knew these. Perhaps we should just keep this as is (add the recipient seed and address as you did), but with unused addresses and keys. Then, if we don't say so yet, instruct developers to set `.env.test` (or is it `.env.local`) when developing the gem.